src/reanimated2/core.ts

```diff
@@ -248,11 +248,13 @@ export function enableLayoutAnimations(
 export function configureLayoutAnimations(
   viewTag: number,
   type: string,
-  config: LayoutAnimationFunction | Keyframe
+  config: LayoutAnimationFunction | Keyframe,
+  sharedTransitionTag = ''
```

Comment:
ok, there is a default value here so I don't think we need one in the NativeReanimatedModule

Reply:
I don't like the the fact that regular layout animations will receive empty string as `sharedTransitionTag`, can we use `undefined` instead?